neurons/miners/bitcoin/funds_flow/graph_search.py

```diff
@@ -51,12 +51,11 @@ def execute_query(self, query: protocol.Query) -> protocol.QueryOutput:
             return result
         except Exception as e:
             raise Exception(f"cypher query execution error: {e}")
-    
-    
+
     def execute_cypher_query(self, cypher_query: str):
         with self.driver.session() as session:
             result = session.run(cypher_query)
-            return result
+            return result.single()
```

Comment:
If memgraph returns multiple results, it should arise warnings. It may cause issue.

Reply:
ok, i added new method ```execute_benchmark_query```, idea is that benchmat must return single scalar value;
as we use those in Kmeans clustering to determinate majority of results

Reply:
check new commit